api/web3server_utils.go

```diff
@@ -87,11 +87,19 @@ func ethAddrToIoAddr(ethAddr string) (address.Address, error) {
 	return address.FromHex(ethAddr)
 }
 
+func IoAddrToEvmAddr(ioAddr string) (common.Address, error) {
+	address, err := address.FromString(ioAddr)
+	if err != nil {
+		return common.Address{}, output.NewError(output.ConvertError, "", err)
```

Comment:
I looked inside ioctl/output and wasn't sure if i could easily replace 
`output.NewError(output.ConvertError, "", err)`
with a function from github.com/pkg/errors.
So, I just kept it here and imported "github.com/iotexproject/iotex-core/ioctl/output".

Are we trying to move away from using stuff in the ioctl directory? Or was it specifically ioctl/utils that we want to avoid?